### PR TITLE
release: Fix git commit by adding user and email

### PR DESCRIPTION
**Pull request by marclop**
## Description

Configures the git username and email through `git config` command since
the environment variables seem to have no effect on the release target.

## Motivation and Context

Broken release GitHub action.

### CODE REVIEW

File: scripts/update-brew-tap.sh [hunk 1]
```diff
@@ -29,7 +29,8 @@ cd /tmp/homebrew-tap
 hub fork --no-remote
 hub remote add fork https://github.com/${GITHUB_USER}/homebrew-tap
 hub add Formula/ecctl.rb
-hub checkout -b f/update-ecctl-formula-to-${VERSION}
```
Comment (karencfv) [inline scripts/update-brew-tap.sh:32]: why was this line removed?

Comment (marclop) [reply scripts/update-brew-tap.sh:32]: Good catch, it shouldn't have

**Review by karencfv — APPROVED** [review]
Looks good!
